CONTRIBUTING.md

```diff
@@ -22,13 +22,70 @@ The project has a mailing list and two discussion channels in IRC:
 Please avoid emailing maintainers found in the MAINTAINERS file directly. They
 are very busy and read the mailing lists.
 
-## Getting Started
+## Issues
+
+When filing an issue, make sure to answer these questions in the issue body:
```

Comment:
Can we just make this a [template](https://github.com/blog/2111-issue-and-pull-request-templates) instead? as @tmrts says, the less contributors have to do, the better